etc/config.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <dev>
+            <js>
+                <minify_exclude>
+                      pub/static/adminhtml/Magento/backend/en_US/CloudFlare_Plugin/js/compiled.min.js
```

Comment:
tried this with "compiled", "compiled.js", "compiled.min.js" none of them worked

Reply:
I tried a ton of combinations of excluding file names with/without paths and extensions. none of it seemed to work. also tried putting config.xml inside of adminhtml folder with no success...

Reply:
Will look in more details. Thanks for testing.